google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java

```diff
@@ -97,16 +98,22 @@ public void deploy(
     if (environment.isStandard() || environment.isFlexCompat()) {
       appYamlName = FlexibleFacetEditor.APP_YAML_FILE_NAME;
     } else {
+      String moduleName = deploymentConfiguration.getModuleName();
```

Comment:
agreed - added a todo. this is something we should do soon: https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1505